### PR TITLE
Finished the dropdown menu

### PR DESCRIPTION
**Pull request by m-khezri**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,18 +13,15 @@
...
-        <button type="button" class="btn btn-outline-info my-auto" id="themeButton">Select Theme <i class="fa fa-eyedropper"></i></button>
+    <nav class="bg-light w-100 mx-auto px-5 d-flex justify-content-center shadow sticky-top">  
+        <img class="m-3" src="./img/logo.png" alt="Logo">          
+          <div class="input-group mx-5 my-auto">
+            <input type="text" placeholder="Enter your message" class="form-control" aria-label="Text input with dropdown button" id="message-input">
+                <div class="input-group-append" id="users-select">
+                      <!-- Populate users here from an array -->
+                </div>
+           </div>
+        <button type="button" class="btn btn-outline-info my-auto" id="themeButton">Select Theme <i class="fa fa-eyedropper"></i> </button>
```
Comment (mjleavell) [inline index.html]: change id to theme-button for consistency

File: javascripts/components/chattyNav.js [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
\ No newline at end of file
+const userPopulate = () => {
+  let domString = '';
+  domString += `<select class="selectpicker btn-success">`;
+  domString += `<option class="FirstItemHide">Select User</option>`;
+  for (let i = 0; i < users.length; i++) {
+    domString += `<option class="bg-light text-dark my-4" id="${users[i]}-user">${users[i]}</option>`;
+  };
+  domString += `</select>`;
+
```
Comment (mjleavell) [inline javascripts/components/chattyNav.js]: delete this extra line

File: javascripts/main.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import { getMessageData } from './data/startupData.js';
+import { userPopulate } from './components/chattyNav.js';
 import { clearMsgEvent } from './events/navbarEvents.js';
 
+
```
Comment (mjleavell) [inline javascripts/main.js:4]: delete this extra line

**Comment by mjleavell** [comment]
@m-khezri Navbar is working great! I made a few minor comments, regarding formatting (see above). Once those changes are made, I'll give you the thumbs up

**Comment by MCrank** [comment]
Outside of @mjleavell comments looks good to me.
👍 🦄

File: index.html (hunk 1 above)
Comment (m-khezri) [reply index.html]: done

File: javascripts/components/chattyNav.js (hunk 2 above)
Comment (m-khezri) [reply javascripts/components/chattyNav.js]: done

File: javascripts/main.js (hunk 3 above)
Comment (m-khezri) [reply javascripts/main.js:4]: done

File: javascripts/components/chattyNav.js [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
+const users = ["Mohammad", "Lance", "Marco", "Maggie"];
+
+const userPopulate = () => {
+  let domString = "";
+  domString += `<select class="selectpicker btn-success">`;
+  domString += `<option class="FirstItemHide">Select User</option>`;
+  for (let i = 0; i < users.length; i++) {
+    domString += `<option class="bg-light text-dark my-4" id="${
+      users[i]
+    }-user">${users[i]}</option>`;
```
Comment (mjleavell) [inline javascripts/components/chattyNav.js]: @m-khezri can you put all of this on one line? 
domString += `<option class="bg-light text-dark my-4" id="${users[i]}-user">${users[i]}</option>`;

Comment (m-khezri) [reply javascripts/components/chattyNav.js]: When I save the file VS-Code automatically arrange the code like this!! This is due to the extension Marco recommended to auto arrange codes after saving, Let me fix it...

Comment (mjleavell) [reply javascripts/components/chattyNav.js]: ha it's okay then. it will probably get reformatted again. i think it's fine!

**Comment by mjleavell** [comment]
👍 👍 👍 👍 👍 👍

**Pull request merged**